extensions/amp-bind/0.1/bind-evaluator.js

```diff
@@ -79,6 +80,22 @@ export class BindEvaluator {
   }
 
   /**
+   * Removes all parsed bindings for the provided expressions.
+   * @param {!Array<string>} expressionStrings
+   */
+  removeBindingsWithExpressionStrings(expressionStrings) {
+    const expressionsToRemove = Object.create(null);
+    for (let i = 0; i < expressionStrings.length; i++) {
+      expressionsToRemove[expressionStrings[i]] = true;
+    }
+
+    this.parsedBindings_ = filterSplice(this.parsedBindings_, binding => {
+      const expressionString = binding.expression.expressionString;
+      return expressionsToRemove[expressionString];
```

Comment:
You want `!` this. `#filterSplice` (and `#filter`) keep the items that return true and remove the items that return false.